Doc/library/functions.rst

```diff
@@ -1718,6 +1718,12 @@ are always available.  They are listed here in alphabetical order.
 
    See also :ref:`bltin-type-objects`.
 
+   Keyword arguments provided to the three argument form are passed to the
+   appropriate metaclass machinery in the same way that keywords in a class
```

Comment:
IMO, appropriate metaclass machinery seems slightly nebulous and a user might have to search up or guess how passing keywords to classes work. Perhaps include a mention of ``object.__init_subclass__`` somewhere?
```suggestion
   appropriate metaclass machinery (usually :meth:`~object.__init_subclass__`)
   in the same way that keywords in a class
```